src/views/tutorial-view/utils/mdx-components.tsx

```diff
@@ -66,6 +67,7 @@ const MDX_COMPONENTS = {
 	p: MdxP,
 	table: MdxTable,
 	blockquote: MdxBlockquote,
+	TryHcpCallout: MdxTryHcpCallout,
```

Comment:
Just some naming thoughts, I do like this capitlization better, but wanted to make sure I bring it up. https://github.com/hashicorp/vault/pull/18080#discussion_r1029760192

Reply:
☝️ Great call, I'm not sure on the authoring preference 🤔 I'm not sure who'd be best to make this call.

Re: adjusting later, once it's in use, it would be hard to deprecate... I think we could expose it under both capitalization variants if we want though - not sure if that might be helpful, or at least helpful enough to offset the messiness? Eg:

```suggestion
	TryHcpCallout: MdxTryHcpCallout,
	TryHCPCallout: MdxTryHcpCallout,
```

Reply:
Added [a poll in Slack](https://hashicorp.slack.com/archives/C01KCU4HDPY/p1669155212933379) - will wait for some responses to roll in before merging this PR 👍 (Or if no responses... I guess we can make a call or something!)